spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +75,21 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	public void setArguments(Map<Integer, Map<String, String>> arguments) {
+		this.arguments = arguments;
+	}
+
+	public Map<Integer, Map<String, String>> getArguments() {
+		return arguments;
+	}
+
+	public List<ModuleLaunchRequest> asModuleLaunchRequests() {
```

Comment:
Sorry, `ModuleLauncher` would not be the right place; I meant *`ModuleLauncherRunner`*. I agree that `ModuleLauncher` should take the `ModuleLaunchRequest` objects, but I think this class should be for *properties*, not construction of those launch request objects.